### DISCUSSION TITLE
Why several jobs added with same priority will be processed in LIFO instead of FIFO

### DISCUSSION THREAD

**Discussion by guilhermebkel** [taskforcesh/bullmq#1716] [Q&A] [answered]
Hey, I just gave a look into the prioritized jobs docs and found a very specific info:

> If several jobs are added with the same priority value, then the jobs within that priority will be processed in LIFO (Last in first out) fashion.

So I have the following doubts:
- Is there any specific reason for this behavior?
- Is there any place in the code I can see this business rule being applied?

**Comment by manast** [comment]
Hmm, I updated that documentation recently, It could just have been me being tired and switching FIFO with LIFO, I will double-check.

**Comment by manast** [comment, marked as answer]
Yes, I think I miswrote the queue policy, it is FIFO and not LIFO, fixing it now.
https://github.com/taskforcesh/bullmq/blob/master/src/commands/includes/addJobWithPriority.lua
